compiler/optimizer/pom.xml

```diff
@@ -45,6 +45,11 @@ under the License.
         <groupId>org.apache.nemo</groupId>
         <artifactId>nemo-conf</artifactId>
         <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.nemo</groupId>
+        <artifactId>nemo-runtime-common</artifactId>
```

Comment:
compiler depends on runtime common?

Reply:
Compiler runs on the driver (which uses the runtime-common module), and needs access to the metrics with the changes now..